app/components/clipboard_button_component.css.scss

```diff
@@ -1,4 +1,5 @@
-@use 'uswds-core' as *;
+@use 'required' as *;
```

Comment:
I'd like to find a better way to do this, since having some files with `@use 'uswds-core' as *;` and others with `@use 'required' as *;` is going to cause confusion, but Sass compilation will error when trying to load both `uswds-core` and `required` as separate directives.

Reply:
b8b4f67 is a slight improvement at least as far as restoring `@use 'uswds-core' as *;` as the conventional way to bring USWDS functions in scope.

It might be more streamlined to go the opposite direction, creating a separate "core" entrypoint that we maintain (basically what `required` is), and change every stylesheet to use that instead. Then, we still have consistency, while only needing one `@use 'core' as *;`, vs. separate `@forward 'required'; @use 'uswds-core' as *;`. The downside is this moves away from how someone might expect to use USWDS. It could be interesting to try to add another layer of "interception" like what LGDS does, so that `@use 'uswds-core' as *;` is actually loading a stylesheet internal to the IdP.

Reply:
> It could be interesting to try to add another layer of "interception" like what LGDS does, so that `@use 'uswds-core' as *;` is actually loading a stylesheet internal to the IdP.

Gave this a try in 316cfd431. The hard part was figuring out how to load the "original" `uswds-core`, but I managed it by adding `node_modules` as a separate load path and referencing the `@18f/identity-design-system` path from there.

This _might_ create some possible performance impact depending if more load paths causes a lot of filesystem noise, since we only need to resolve `node_modules` in this one location, but now it'll be the first thing checked for every `@forward`, `@use`, and `@import`.

Reply:
> This _might_ create some possible performance impact depending if more load paths causes a lot of filesystem noise, since we only need to resolve `node_modules` in this one location, but now it'll be the first thing checked for every `@forward`, `@use`, and `@import`.

Apparently I forgot that `node_modules` is a load path by default 😅 

https://github.com/18F/identity-idp/blob/c58e36870403b6c941beca77f2287c5f78c859ea/app/javascript/packages/build-sass/index.js#L34